app/services/folio/types.rb

```diff
@@ -46,42 +46,42 @@ def circulation_rules
     end
 
     def service_points
-      get_type('service_points').map { |p| Folio::ServicePoint.from_dynamic(p) }.index_by(&:id)
+      load_cache(type: 'service_points').map { |p| Folio::ServicePoint.from_dynamic(p) }.index_by(&:id)
     end
 
     def policies
       @policies ||= {
-        request: get_type('request_policies').index_by { |p| p['id'] },
-        loan: get_type('loan_policies').index_by { |p| p['id'] },
-        overdue: get_type('overdue_fines_policies').index_by { |p| p['id'] },
-        'lost-item': get_type('lost_item_fees_policies').index_by { |p| p['id'] },
-        notice: get_type('patron_notice_policies').index_by { |p| p['id'] }
+        request: load_cache(type: 'request_policies').index_by { |p| p['id'] },
+        loan: load_cache(type: 'loan_policies').index_by { |p| p['id'] },
+        overdue: load_cache(type: 'overdue_fines_policies').index_by { |p| p['id'] },
+        'lost-item': load_cache(type: 'lost_item_fees_policies').index_by { |p| p['id'] },
+        notice: load_cache(type: 'patron_notice_policies').index_by { |p| p['id'] }
       }
     end
 
     # rubocop:disable Metrics/AbcSize
     def criteria
       @criteria ||= {
-        'group' => get_type('patron_groups').index_by { |p| p['id'] },
-        'material-type' => get_type('material_types').index_by { |p| p['id'] },
-        'loan-type' => get_type('loan_types').index_by { |p| p['id'] },
-        'location-institution' => get_type('institutions').index_by { |p| p['id'] },
-        'location-campus' => get_type('campuses').index_by { |p| p['id'] },
+        'group' => load_cache(type: 'patron_groups').index_by { |p| p['id'] },
+        'material-type' => load_cache(type: 'material_types').index_by { |p| p['id'] },
+        'loan-type' => load_cache(type: 'loan_types').index_by { |p| p['id'] },
+        'location-institution' => load_cache(type: 'institutions').index_by { |p| p['id'] },
+        'location-campus' => load_cache(type: 'campuses').index_by { |p| p['id'] },
         'location-library' => libraries,
         'location-location' => locations
       }
     end
     # rubocop:enable Metrics/AbcSize
 
     def libraries
-      @libraries ||= get_type('libraries').index_by { |p| p['id'] }
+      @libraries ||= load_cache(type: 'libraries').index_by { |p| p['id'] }
     end
 
     def locations
-      @locations ||= get_type('locations').index_by { |p| p['id'] }
+      @locations ||= load_cache(type: 'locations').index_by { |p| p['id'] }
     end
 
-    def get_type(type)
+    def load_cache(type:)
```

Comment:
I personally like the current name more than `load_cache`. I also don't know why we want a keyword arg here.

Reply:
Oops. I thought it looked OK but happy to defer / discuss.

Reply:
`load_cache` tells me what the method is doing. It's populating a cache from the files on disk.  `get_type` on the other hand is two very ambigous words. "get" from where?  "type" of what?

Reply:
I don't think being cached is the important part of this. We're just trying to load FOLIO type data.